### ISSUE TITLE
Add mobs to guard loots

### ISSUE THREAD

**Issue by Arckenver** [Arckenver/MightyLoot#4]
Add ability to spawn mobs with loots so they protect them, and so make them harder to get.

**Comment by Arckenver** [comment]
We could add a node in lootConfig like this

```
mobs {
    Zombie=5
    Skeleton=3
    Spider=1
}
```

**Comment by Arckenver** [comment]
Fixed as of commit fb5ed5c38e19c6776c4ee0612c62daa8fa577950.

**Comment by Arckenver** [comment]
To add loot guardians, add these nodes in config file

```
options {
    lootGuardians {
        Zombie=5
        Skeleton=3
    }
}
```

Specify mob type using official [`EntityTypes`](http://minecraft.gamepedia.com/Data_values/Entity_IDs) and the number of entities of this type to make spawn.
